v2/swagger.yaml

```diff
@@ -434,12 +491,26 @@ definitions:
           type: string
       ecosystem:
         type: string
-      security:
+      public_vulnerabilities:
+        type: array
+        items:
+          $ref: "#/definitions/Vulnerability"
+        description: Publicly known vulnerability details
+      private_vulnerabilities:
         type: array
+        items:
+          $ref: "#/definitions/Vulnerability"
+        description: Private vulnerability details, available only to registered users
+      affected_direct_deps:
```

Comment:
It is an existing field which is already used in production, but not present in the document.